tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+DATA_FOLDER = Path("tests/golden_data").resolve()
```

Comment:
This will block you from running tests outside the main repository folder. Use the same strategy we use in haddock3:

https://github.com/haddocking/haddock3/blob/1555fd00e3fd4f70ccbb406b7b0ed9d6aeeca1f1/tests/__init__.py#L5-L6